webview-ui/src/KaitoModels/KaitoModels.tsx

```diff
@@ -410,3 +357,46 @@ inference:
         </>
     );
 }
+
+// exported function to be shared in other kaito-related webviews
+export function generateYAML(model: string): [string, string | undefined] {
```

Comment:
🧋minor suggestion, I know this is generic info anyways, I wonder if we rename this to `generateKaitoYAML` or more appropriate self explanatory name, please? what do you think